tracer/test/Datadog.Trace.Security.Unit.Tests/FuzzEncoder.cs

```diff
@@ -31,8 +31,16 @@ public FuzzEncoder(ITestOutputHelper outputHelper)
         [Fact]
         public void LetsFuzz()
         {
+#if _NET5_0_OR_GREATER_
             // if we don't throw any exceptions and generate a valid object the the test is successful
-
+            for (int x = 0; x < 1000; x++)
```

Comment:
What's this all about?

Reply:
My bad 🙈 
Fixed in 550e98bb30b9ab2cd9d8b847a1ce5559128e04e1